src/protocols/ethereum/components/TransferSendForm.vue

```diff
@@ -276,6 +287,8 @@ export default defineComponent({
     );
 
```

Comment:
`getTransferGasLimit` is still calculating using only `ethActiveNetworkSettings` a not protocol specific,